WeScan/Edit/EditScanViewController.swift

```diff
@@ -36,6 +36,13 @@ final class EditScanViewController: UIViewController {
         button.tintColor = navigationController?.navigationBar.tintColor
         return button
     }()
+    
+    lazy private var cancelButton: UIBarButtonItem = {
+        let title = NSLocalizedString("wescan.edit.button.cancel", tableName: nil, bundle: Bundle(for: EditScanViewController.self), value: "Cancel", comment: "A generic cancel button")
```

Comment:
We don't directly use `NSLocalizedString`. Search `Localizable` protocol in the code. you will find examples how we use it.